libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -454,6 +454,16 @@ class BaseChatOpenAI(BaseChatModel):
     """Total probability mass of tokens to consider at each step."""
     max_tokens: Optional[int] = Field(default=None)
     """Maximum number of tokens to generate."""
+    reasoning_effort: Optional[str] = None
```

Comment:
should we type as literal?

Reply:
doesn't seem super high value? esp if we care about being forwards compatible